src/command/dev/router/config.rs

```diff
@@ -235,12 +235,12 @@ impl RouterConfigReader {
 
                 // disable the health check unless they have their own config
                 if input_yaml
-                    .get("health-check")
+                    .get("health_check")
```

Comment:
I think we should also keep the old check around as well, 

If NEITHER `health-check` NOR `health_check` are specified in the router config, THEN insert `health_check.enabled: false`.

This prevents a case where we don't detect a `health_check` and so we try to insert our own, but now it exists side by side with a user specified `health-check`.

Reply:
Good call, fixed! I used `or_else` instead since I think it does the same thing with less duplication.

Reply:
oh sick i love learning about new combinators